src/modules/keyboardmanager/ui/EditShortcutsWindow.cpp

```diff
@@ -144,13 +144,13 @@ void createEditShortcutsWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMa
 
     //  Text block for information about remap key section.
     TextBlock shortcutRemapInfoHeader;
-    shortcutRemapInfoHeader.Text(L"Select shortcut you want to change (Original Shortcut) and the shortcut (New Shortcut) you want it to invoke.");
+    shortcutRemapInfoHeader.Text(L"Select shortcut you want to change (Shortcut) and the shortcut (Mapped To) you want it to invoke.");
```

Comment:
Should we put (Mapped To) at the end like done in the case of Remap Keyboard or do you think leaving it like this is better?